src/main/java/duke/exceptions/NoSuchCommandException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exceptions;
+
+public class NoSuchCommandException extends Exception{
```

Comment:
Great use of creating your own exception to deal with errors in the code. Perhaps instead of using `super(message);`, you could print "Invalid command" here and append any additional information after it, since I noticed that most of the messages specified were "Invalid command".